runtime/onert/api/src/nnfw_api_internal.h

```diff
@@ -168,7 +168,6 @@ struct nnfw_session
   State _state{State::INITIALIZED};
   std::shared_ptr<onert::ir::Subgraphs> _subgraphs;
   std::unique_ptr<onert::compiler::CompilerOptions> _coptions;
-  std::unique_ptr<onert::compiler::Compiler> _compiler;
```

Comment:
`_compiler` is gone! It is not necessary to keep `_compiler` as a member `nnfw_session`. It is only used and should be live during `compile()` phase. Let it be a local variable of `compiler()`.